### DISCUSSION TITLE
Use an other instance of axios ?

### DISCUSSION THREAD

**Discussion by vince83110** [nuxt-community/auth-module#1505] [Q&A]
I have my NuxtJS app running using ElasticSearch as a data store.

I have configurated my Nuxt axios instance to use the ElasticSearch (SSR and client).

I want to keep this configuration, is it possible to make the auth module to not use my Nuxt axios instance ? But an other one ?

I noticed Nuxt auth changes my headers, and I need basic auth for my ElasticSearch.

I tried to redefine the axios in a plugin: 

```
export default ({ app }) => {
  app.$auth.ctx.$axios = app.$axiosApi
}
```

But no success. Can I override the constructor or something ? Thanks.

**Comment by avxkim** [comment]
Just make your own auth, much easier. This is not customizable, for example, if you want to send different headers while getting token - it's not possible to do so.

**Comment by AllanOricil** [comment]
@vince83110  It is a good idea, but I don't think they will change it now to help you. So it is easier if your app is the one that creates a new instance of axios. 
Use this to guide you

https://stackoverflow.com/questions/67718493/nuxtjs-manage-several-axios-instances
